### ISSUE TITLE
2.1 variant without amp

### ISSUE THREAD

**Issue by tbrasser** [sonocotta/esparagus-echo#2]
For my 2.1 active speaker/sub setup I'd love a "streamer" variant of these

**Comment by tbrasser** [comment]
Also, this may serve as va firmware inspiration: https://github.com/esphome/voice-kit

(I'm currently using gnumpi's custom components @ 48kHz on my onju-voice's)

**Comment by anabolyc** [comment]
Hey @tbrasser I also designed [Hifi esparagus](https://github.com/sonocotta/esparagus-media-center?tab=readme-ov-file#esparagus-hifi-medialink) that is a line level device. It doesn't have a mic, but if you're looking for media player device, it should nail it.

Thanks for the link. I'm working on it, it is not very straightforward :)

**Comment by tbrasser** [comment]
Specifically looking for one with a mic. Currently running an rpi4 with snapclient + hifiberry digital spdif to a dac that can do 2.1.

Would be amazing to use esphome/esp32 for this purpose.

**Comment by anabolyc** [comment]
I don't have anything like it available. As I mentioned, I'm planning to make a dual TAS5805 board in the future, I can place a MIC on the board (or simply a header for the off-the-shelf module, which would cost me nothing) but this is still a work in progress.

Can you please describe your setup and how you're planning to use it?

**Comment by anabolyc** [comment]
By the way, [Louder-ESP32](https://www.tindie.com/products/sonocotta/louder-esp32/) with the Header Mic is already available: 

<img width="936" height="608" alt="Image" src="https://github.com/user-attachments/assets/44c075fc-6ce2-43ba-93b4-c8a5bfcf840f" />

Updated [HiFi-ESP32](https://www.tindie.com/products/sonocotta/hifi-esp32/) also came from the factory:

<img width="896" height="672" alt="Image" src="https://github.com/user-attachments/assets/3e6f24a9-416e-4f58-acdb-337a2c6e7f6f" />

**Comment by tbrasser** [comment]
Nice! Can the hifi do 2.1?

**Comment by anabolyc** [comment]
Not really. But I'm working on the PCM5122-based device, sort of HiFi-Plus
- PCM5122 have built-in DSP, so you can apply filters (BQ type) inside DAC 
- You can connect mutiple DACs in master-slave mode, so one DAC is driving satellies and send subwoofer signal to the next one
- So you can build multi-channel fairly easily, on paper at least. As usual, TI tooling is quite poor, so it would require extensive research, but it looks really promising.
